docs/sponsorship/slack.rst

```diff
@@ -0,0 +1,75 @@
+Slack Network Sponsorship
+=========================
+
+Write the Docs is a community focused on all things related to software documentation.
+Our :doc:`Slack network </slack>` is the beating heart of our community,
+where thousands of documentarians hang out regularly to chat.
+Sponsorship of our Slack network does two things:
+
+* It allows us to cover the cost of moderating and managing our Slack community
+* It allows you to get your name and message in front of our audience of :doc:`/documentarians`
+
+You can join our Slack and read some of the messages to get a feel for the community.
+
+Audience
+--------
+
+Our audience is made up of technical writers and decision makers in the software documentation industry.
+The current Slack stats are:
+
+* Over 17,000 users
+* 100+ channels
+
+We don't have specific demographic data for our Slack,
+but our conferences are made up of:
+
+- Technical Writers (60%)
+- Developers (10%)
+- Support Staff (10%)
+- Managers (10%)
+- Community Contributors, Enthusiasts & Other Folks (10%)
+
+
+Sponsored entry format
+----------------------
+
+Sponsor copy will be posted to the ``#announcements`` channel in Slack,
+which is joined by every user in our Slack.
+It's used for community-wide announcements,
+so your message will be very visible in the community.
+
+The current format is:
+
+* Up to 300 characters of copy
+* Up to 2 links in that copy
```

Comment:
Do link chars contribute to character count? If so we should explicitly state that link shorteners are OK in this context.

Reply:
Occurs to me that Slack now supports inline links so maybe this doesn't matter

Reply:
Yea, link text would count, but not the actual link I think 👍

Reply:
```suggestion
* Up to 2 links in that copy - we generally recommend inline links, so that some of your text is linked.
```